<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rahul_P" name="Invalid_Emailid" tests="18" failures="1" timestamp="5 Sep 2014 13:12:12 GMT" time="1122.422" errors="0">
  <testcase name="invalid_emailid" time="29.922" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="30.159" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="27.966" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="28.677" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="39.72" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="28.068" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="40.422" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="34.844" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="7.333" classname="com.linkedin.cases.CreateNewAccount">
    <failure type="org.openqa.selenium.remote.SessionNotFoundException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;2.42.2&amp;apos;, revision: &amp;apos;6a6995d&amp;apos;, time: &amp;apos;2014-06-03 17:42:03&amp;apos;
System info: host: &amp;apos;Rahul_P&amp;apos;, ip: &amp;apos;192.168.56.1&amp;apos;, os.name: &amp;apos;Windows 8.1&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.3&amp;apos;, java.version: &amp;apos;1.7.0_55&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.SessionNotFoundException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '2.42.2', revision: '6a6995d', time: '2014-06-03 17:42:03'
System info: host: 'Rahul_P', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'x86', os.version: '6.3', java.version: '1.7.0_55'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:147)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:59)
	at org.openqa.selenium.support.pagefactory.AjaxElementLocator.access$0(AjaxElementLocator.java:1)
	at org.openqa.selenium.support.pagefactory.AjaxElementLocator$SlowLoadingElement.isLoaded(AjaxElementLocator.java:140)
	at org.openqa.selenium.support.ui.SlowLoadableComponent.get(SlowLoadableComponent.java:48)
	at org.openqa.selenium.support.pagefactory.AjaxElementLocator.findElement(AjaxElementLocator.java:70)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:34)
	at com.sun.proxy.$Proxy11.sendKeys(Unknown Source)
	at com.linkedin.pages.indexpages.CreateNewAccount_IndexPage.verify_emailid(CreateNewAccount_IndexPage.java:361)
	at com.linkedin.cases.CreateNewAccount.invalid_emailid(CreateNewAccount.java:168)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- invalid_emailid -->
  <testcase name="invalid_emailid" time="35.296" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="27.993" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="40.936" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="37.033" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="28.016" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="41.928" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="37.279" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="28.583" classname="com.linkedin.cases.CreateNewAccount"/>
  <testcase name="invalid_emailid" time="30.957" classname="com.linkedin.cases.CreateNewAccount"/>
</testsuite> <!-- Invalid_Emailid -->
